util/topgen/lib.py

```diff
@@ -21,6 +21,47 @@
 from .intermodule import get_dangling_im_def # noqa : F401 # isort:skip
 
 
+class Name(object):
+    """We often need to format names in specific ways; this class does so."""
```

Comment:
Can you mention the parts functionality as well?

Reply:
Yes, I've expanded this documentation sting a bit.